hw/top_earlgrey/data/top_earlgrey_pkg.sv.tpl

```diff
@@ -1,6 +1,6 @@
-// Copyright lowRISC contributors.
-// Licensed under the Apache License, Version 2.0, see LICENSE for details.
-// SPDX-License-Identifier: Apache-2.0
+## Copyright lowRISC contributors.
```

Comment:
SV template requires C style comments. Why change this?

Reply:
This is a "mako-style" comment. It will be stripped when rendering the template. I changed that since I prepend the header + command info when writing out the rendered file.

In hindsight however it might be easier to just leave it as is and insert the command info using a mako variable, since at the moment all `sv.tpl` files are checked for `//` style comments by `licence-checker.py`...

Reply:
You need to ensure that both the template and the resulting generated file have a license header. The `//` comments should achieve this goal, but any other way to do the same is equally ok.

Reply:
Yep, I fixed this by leaving the `//` style comments as they where and adding a `${gencmd}` mako variable that prints the generate command into the template (after the header).